docs/spec/auth/jwt.md

```diff
@@ -0,0 +1,347 @@
+<!--[metadata]>
++++
+title = "Token Authentication Implementation"
+description = "Describe the reference implementation of the Docker Registry v2 authentication schema"
+keywords = ["registry, on-prem, images, tags, repository, distribution, JWT authentication, advanced"]
+[menu.main]
+parent="smn_registry_ref"
++++
+<![end-metadata]-->
+
+# Docker Registry v2 Bearer token specification
+
+This specification covers the `docker/distribution` implementation of the
+v2 Registry's authentication schema.  Specifically, it describes the JSON
+Web Token schema used to implement the opaque Bearer tokens used for
+authentication.
+
+This document borrows heavily from the [JSON Web Token Draft Spec](https://tools.ietf.org/html/draft-ietf-oauth-json-web-token-32)
+
```

Comment:
It might be good to call out the role of this protocol here. Specifically, that it is between the auth server and the registry and should be opaque to the client.